cluster-service/Makefile

```diff
@@ -20,13 +21,16 @@ deploy:
 			-n clusters-service \
 			--query clientId) && \
 	CS_SERVICE_PRINCIPAL_CREDS_BASE64='$(shell az keyvault secret show --vault-name "service-kv-aro-hcp-dev" --name "aro-hcp-dev-sp-cs" | jq .value -r | base64 | tr -d '\n')' && \
+	TENANT_ID=$(shell az account show --query tenantId --output tsv) && \
 	oc process --local -f deploy/openshift-templates/arohcp-service-template.yml \
 	  -p AZURE_CS_MI_CLIENT_ID=$${AZURE_CS_MI_CLIENT_ID} \
+	  -p TENANT_ID=$${TENANT_ID} \
 	  -p REGION=${REGION} \
+	  -p KEYVAULT_NAME=${KEYVAULT_NAME} \
```

Comment:
should this be named `SERVICE_KEYVAULT_NAME` to match the parameter name expected by the template?

Reply:
good catch